app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -167,6 +168,23 @@ def log
 
 private
 
+  def validate_relevant_collection_window
+    return unless start_date
+    return unless bulk_upload.form
+
+    unless bulk_upload.form.valid_start_date_for_form?(start_date)
+      errors.add(:field_96, I18n.t("validations.date.outside_collection_window"))
+      errors.add(:field_97, I18n.t("validations.date.outside_collection_window"))
+      errors.add(:field_98, I18n.t("validations.date.outside_collection_window"))
```

Comment:
maybe worth combining these?

Reply:
```suggestion
      %i[field_96 field_97 field_98].each { |field| errors.add(field, I18n.t("validations.date.outside_collection_window")) }
```

Reply:
sorry, i think this is less read-able and requires more brain power so gonna leave